src/cudamatrix/cu-device.h

```diff
@@ -26,6 +26,7 @@
 #if HAVE_CUDA == 1
 
 #include <cublas_v2.h>
+#include <cudnn.h>
```

Comment:
They told me privately that they would consider amending the license to
allow us to redistribute it.


On Mon, Mar 28, 2016 at 6:04 PM, Daniel Galvez <notifications@github.com>
wrote:

> In src/cudamatrix/cu-device.h
> <https://github.com/kaldi-asr/kaldi/pull/604#discussion_r57641729>:
>
> > @@ -26,6 +26,7 @@
> >  #if HAVE_CUDA == 1
> >
> >  #include <cublas_v2.h>
> > +#include <cudnn.h>
>
> Installation is not easy. You need to register as a developer on NVIDIA's
> website, and you cannot redistribute the binary: "NVIDIA hereby grants
> Licensee the following limited, non-exclusive, *non-transferable*,
> non-sublicensable right to use the SOFTWARE"
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/604/files/b1e82b2faceed80758f8fcb33e8ba203057d8252..722c7e5a927305e2e02ba6735a2c14761fed6d05#r57641729>
>